README.md

```diff
@@ -11,4 +11,34 @@ the folder 3rdparty.
 
 Screenshots
 -----------
-![Alan Wakes American Nightmare 1](screenshots/awan1.png)
\ No newline at end of file
+![Alan Wakes American Nightmare 1](screenshots/awan1.png)
+
+
+Building and running on Linux
+--------
+
+Make sure to not include the `$` when running these. It signifies that the commands should be run as your user, not root.
+
+```bash
+$ git clone https://github.com/OpenAWE-Project/OpenAWE.git --recurse-submodules
+$ cd OpenAWE
+$ mkdir build
+$ cd build
+```
+
+The following cmake invocation will fail if you don't have all necessary dependencies installed on your host system.
```

Comment:
It would be nice to have a list of necessary dependencies at that point. These are everything which is retrieved in cmake by find_package() with the option required plus entt and spdlog. It should also be mentioned, that entt and cxxopts can be used as a system distribution or as delivered by the submodules. There are cmake options for this (USE_SYSTEM_*).